samples/subsys/usb/webusb/README.rst

```diff
@@ -1,21 +1,20 @@
-Title: WebUSB Enabled Custom Driver and A Sample Application
+.. _webusb-sample:
 
-This project includes:
+=========================
+WebUSB sample application
+=========================
 
-1. WebUSB Enabled Custom Driver:
---------------------------------
-
-This enables WebUSB support to the device via the platform capability
-descriptors.
+WebUSB Documentation
+--------------------
 
 For a deeper dive into the WebUSB, refer to
 https://github.com/WICG/webusb/blob/gh-pages/explainer.md
 
 WebUSB API Specification:
 https://wicg.github.io/webusb/
 
-2. Sample Application:
-----------------------
+Overview
+--------
 
 A simple echo application to demonstrate the WebUSB enabled custom
```

Comment:
Make this a complete sentence:

    This simple echo application demonstrates the WebUSB enabled custom class driver.